src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -231,6 +231,7 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
   static styles = css`
     ha-card {
       height: 100%;
+      overflow: scroll;
```

Comment:
```suggestion
      overflow-y: auto;
```
We should not display `x` scrollbar and 'y` should only be displayed if necessary.